airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```
@@ -0,0 +1,348 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data"]
+
+  selector_increment:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data"]
+      
+  selector_users:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data","*"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.pipedrive.com/v1"
+    http_method: "GET"
+    request_parameters:
+      api_token: "{{config['api_token']}}"
```

```suggestion
      api_token: "{{ config['authorization']['api_token'] }}"
```
This is to maintain compatibility with current config.